### ISSUE TITLE
Implement SSR for faster loading

### ISSUE THREAD

**Issue by x5engine** [openintents/app-center#31]
When you load the app it takes a few seconds to render the html and design, that should be done first in the server Server Side Render

https://www.gatsbyjs.org/docs/ssr-apis/

**Comment by friedger** [comment]
Using the material-ui plugin helped already. Closing this now. If you have ideas how to improve the site performance feel free to open a new issue.
